config/version_git.h.in

```diff
@@ -0,0 +1,9 @@
+#define NVIM_VERSION_MAJOR @NVIM_VERSION_MAJOR@
```

Comment:
Should we have include guards around the file?  I noticed config.h doesn't have them either, but it probably should too.

Also, keep in mind folks may be building from a source tarball in the future.  Is version_git.h really what we want to call this file?  It seems a bit of a misnomer, but I'm not sure I have a better suggestion.  version_defines.h?